rules/call_peak.smk

```diff
@@ -1,24 +1,33 @@
+def get_genrich_replicates(wildcards):
```

Comment:
this function becomes ambigous if a biological replicate has the same name as a technical replicate/sample. this is why i split the rule outputs by folder.

If I'm mistaken, or there is a solution, it would be really nice!

Reply:
But you really shouldn't have the same names for those things right? I could add an assertion in configuration. It also becomes unclear in the trackhub which peak/bigwig belongs to what, and in the multiqc report.

Reply:
Or unless you have a single sample for a condition?

Reply:
that would work. I left it as an option, but this would indeed remove possible confusion... go for it!